eclass/kernel-build.eclass

```diff
@@ -270,16 +355,39 @@ kernel-build_merge_configs() {
 	local user_configs=( "${BROOT}"/etc/kernel/config.d/*.config )
 	shopt -u nullglob
 
+	local merge_configs=( "${@}" )
+
+	if [[ ${KERNEL_IUSE_MODULES_SIGN} ]]; then
+		if use modules-sign; then
+			: "${MODULES_SIGN_HASH:=sha512}"
+			cat <<-EOF > "${WORKDIR}/modules-sign.config" || die
+				## Enable module signing
+				CONFIG_MODULE_SIG=y
+				CONFIG_MODULE_SIG_ALL=y
+				CONFIG_MODULE_SIG_FORCE=y
+				CONFIG_MODULE_SIG_${MODULES_SIGN_HASH^^}=y
+			EOF
+			if [[ -e ${MODULES_SIGN_KEY} ]]; then
```

Comment:
Actually, we probably want some additional logic here to allow for pkcs11 URIs

Reply:
Do we want it *now* or do we want it later, after this one is merged? ;-)